rdt/transformers/numerical.py

```diff
@@ -91,16 +124,13 @@ def _learn_rounding_digits(data):
 
         return None
 
-    def fit(self, data):
+    def _fit(self, data):
         """Fit the transformer to the data.
 
         Args:
-            data (pandas.Series or numpy.ndarray):
+            data (pandas.DataFrame or pandas.Series):
                 Data to fit.
         """
-        if isinstance(data, np.ndarray):
```

Comment:
Why is this removed? Is it simply not necessary anymore, or does keeping it make the code crash?

Reply:
I think it was just not necessary.

Reply:
It seems to actually be necessary, since this is what makes the performance tests fail: https://github.com/sdv-dev/RDT/runs/3719384829?check_suite_focus=true#step:5:6430